pkg/setting/setting_plugins.go

```diff
@@ -27,21 +27,27 @@ func extractPluginSettings(sections []*ini.Section) PluginSettings {
 func (cfg *Cfg) readPluginSettings(iniFile *ini.File) error {
 	pluginsSection := iniFile.Section("plugins")
 	cfg.PluginsEnableAlpha = pluginsSection.Key("enable_alpha").MustBool(false)
+	cfg.PluginNavLocation = pluginsSection.Key("nav_location").MustString("")
```

Comment:
@torkelo is this an accidental commit to this PR?  seems unrelated

Reply:
Oops forgot I had some changes going when switching to this PR! Sorry!